### DISCUSSION TITLE
Is the T/D coming?

### DISCUSSION THREAD

**Discussion by Tomtomsen17** [flybywiresim/aircraft#3226] [General]
Do you know when the T/D is coming or will it come?

**Comment by ChruutvoLuzi** [comment]
Eventually, yes for sure, however there wont be any ETAs from FBWSim

**Comment by tracernz** [comment]
Design in-progress https://github.com/flybywiresim/rfcs/blob/000-vertical-navigation/text/000-vertical-navigation.md
Be patient.

**Reply by Tomtomsen17** [reply]
I can’t wait for it

**Reply by robinottawa** [reply]
I've seen it on my dev version lately.

**Reply by ChruutvoLuzi** [reply]
> I've seen it on my dev version lately.

thats the the stock one, it does not work as it should

**Reply by DescendDescend** [reply]
Maybe it would be worth including the Vertical Deviation in the MCDU, the required profile on the PFD, and the MORE DRAG and TOO STEEP PATH calulations as part of the design rather than add them later but maybe these are covered elsewhere?
